lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -1096,6 +1096,7 @@ def show_global_options
               [ 'Prompt', framework.datastore['Prompt'] || Msf::Ui::Console::Driver::DefaultPrompt.to_s.gsub(/%.../,"") , "The prompt string" ],
               [ 'PromptChar', framework.datastore['PromptChar'] || Msf::Ui::Console::Driver::DefaultPromptChar.to_s.gsub(/%.../,""), "The prompt character" ],
               [ 'PromptTimeFormat', framework.datastore['PromptTimeFormat'] || Time::DATE_FORMATS[:db].to_s, 'Format for timestamp escapes in prompts' ],
+              [ 'MeterpreterPrompt', framework.datastore['MeterpreterPrompt'] || (Rex::Compat.is_windows() ? 'meterpreter' : '%undmeterpreter%clr'), 'The meterpreter prompt string' ],
```

Comment:
I don't believe this needs a Windows check, since the standard prompt doesn't have one. `Rex::Text `should take care of any platform quirks when using the `%...` escapes.

Reply:
@acammack-r7 I was trying to maintain the behavior I had observed here: https://github.com/rapid7/metasploit-framework/blob/master/lib/rex/post/meterpreter/ui/console.rb#L29-L33. If you're sure it's unnecessary though, I'd be happy to take it out. I admittedly don't have a windows install to test with.

Reply:
confirmed by applying to a nightly build on Windows 10, the color codes are safe on Windows:
![image](https://user-images.githubusercontent.com/4108654/66653284-49dd3d80-ebfd-11e9-9115-7a688de9186c.png)